From private repo

v-alje opened the pull request.

tysonn commented:
Hi Varun -

This article needs some updates....

1. Remove the in-topic navigation. This week, an automated in-topic navigation will be added to the site. All H2 headings will render in the in-topic navigation, which will appear to the right of the page in wide views or inline in the topic in narrow screen views.

2. As coded, this article won't have a right navigation because it's currently coded with an H1 (#) heading and all the rest are H3 headings (###). The right navigation uses only H2 headings, so I would change all the H3s to H2s (##) so you get the in-topic nav.

3. Replace the bold first sentence with a meaningful H2 (##) that will be the first line in the in-topic navigation.

4. There are some sentence fragments in the "Provision" section.

Thanks,
Tyson

alkohli committed:
Update 5 release notes, apply update 5 guide

alkohli committed:
more changes

alkohli committed:
more changes

alkohli committed:
more changes

alkohli committed:
more changes

alkohli committed:
more changes

alkohli committed:
added images

alkohli committed:
more changes

alkohli committed:
TOC revise for GU5

alkohli committed:
more changes from Mohit and Marcelo

alkohli committed:
more changes from Vijay

alkohli committed:
changes from teddy

alkohli committed:
more changes

alkohli committed:
PR pre-review blocking issues addressed

ShawnJackson committed:
edit pass: security-azure-log-integration

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into security-azure-log-integration

ShawnJackson committed:
edit pass: security-azure-log-integration

ShawnJackson committed:
edit pass: security-azure-log-integration

ktoliver committed:
edit pass: TFS 71944

ktoliver committed:
fix bookmarks

ktoliver committed:
Update hdinsight-troubleshoot-yarn.md

ktoliver committed:
Update hdinsight-troubleshoot-spark.md

ktoliver committed:
Update hdinsight-troubleshoot-yarn.md

ktoliver committed:
Update hdinsight-troubleshoot-spark.md

ktoliver committed:
Update hdinsight-troubleshoot-guide.md

ktoliver committed:
Update hdinsight-troubleshoot-spark.md

ktoliver committed:
Update hdinsight-troubleshoot-hive.md

ktoliver committed:
Update hdinsight-troubleshoot-guide.md

ktoliver committed:
Update hdinsight-troubleshoot-guide.md

ktoliver committed:
Update hdinsight-troubleshoot-hive.md

ktoliver committed:
Update hdinsight-troubleshoot-guide.md

ktoliver committed:
Update hdinsight-troubleshoot-hbase.md

ktoliver committed:
Update hdinsight-troubleshoot-hdfs.md

ktoliver committed:
Update hdinsight-troubleshoot-spark.md

aliuy committed:
Cosmos DB: Event Sourcing Architecture Diagram

aliuy committed:
Cosmos DB event sourcing use-case

aliuy committed:
Update use-cases.md

aliuy committed:
Delete event-sourcing.png

aliuy committed:
Add files via upload

aliuy committed:
Add files via upload

aliuy committed:
fix serverless link

Commit:
change in get-started-linux

ktoliver committed:
Merge branch 'master' into TFS-71944

Commit:
edit pass: application-gateway-customize-waf-rules-cli

aliuy committed:
Merge branch 'master' into master

Commit:
first linux app with Java

Commit:
changes in C#, mac, eclipse doc

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-gateway-customize-waf-rules-batch1

Commit:
edit pass: application-gateway-customize-waf-rules-cli

Commit:
Merge branches 'master' and 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-gateway-customize-waf-rules-batch1

:wq
i# the commit.

Commit:
edit pass: application-gateway-customize-waf-rules-portal

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-gateway-customize-waf-rules-batch1

Commit:
edit pass: application-gateway-customize-waf-rules-portal

Commit:
migration updates

Commit:
fix

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-gateway-customize-waf-rules-batch1

Commit:
edit pass: application-gateway-customize-waf-rules-powershell

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-gateway-customize-waf-rules-batch1

Commit:
edit pass: application-gateway-customize-waf-rules-cli

Commit:
syncing changes

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-gateway-customize-waf-rules-batch1

SnehaGunda committed:
Splitting PS topic

SnehaGunda committed:
Splitting PS topic

SnehaGunda committed:
Splitting PS topic

SnehaGunda committed:
Splitting PS topic

SnehaGunda committed:
updating links

Commit:
some more updates in docs

Commit:
update images

genlin committed:
update the error section

genlin committed:
fix the link issues

Commit:
Migration plan for existing Java applications to work with Maven

Commit:
Addressed PR suggestions

Commit:
some more improvements

Commit:
Minor indentation changes

Commit:
Few small changes to increase score

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into application-gateway-customize-waf-rules-batch1

alkohli committed:
added authentication alert info

Commit:
edit pass: application-gateway-customize-waf-rules-portal

Commit:
edit pass: application-gateway-customize-waf-rules-cli

Commit:
edit pass: application-gateway-customize-waf-rules-powershell

alkohli committed:
fixed warnings

alkohli committed:
fixed warnings

curtand committed:
Acrolinx one file for data-driven improvements

robinsh committed:
Make changes req by Kumail.

Commit:
edit pass: application-gateway-customize-waf-rules-powershell

Commit:
edit pass: application-gateway-customize-waf-rules-powershell

SnehaGunda committed:
updating links

SnehaGunda committed:
updating links

robinsh committed:
more changes per Kumail

SnehaGunda committed:
resolving conflict

SnehaGunda committed:
Updating note

SnehaGunda committed:
Updating links

SnehaGunda committed:
Updating note

SnehaGunda committed:
Updating links

alkohli committed:
minor typos

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into data0821

cjgronlund committed:
AI add

BrucePerlerMS committed:
add note about output formatting and soft-delete status

Commit:
Addressing feedback

BrucePerlerMS committed:
wording

Commit:
edits

cjgronlund committed:
image moved

BrucePerlerMS committed:
clear up language in NOTE

Commit:
Adding assign licenses quickstart

BrucePerlerMS committed:
fix date

Commit:
Making myself author to delete these files.

Commit:
deleted files and put in redirects

BYHAM committed:
SQL DW develop concurrency, formatting changes.

Updating date, moving comments out of code, fixing internal broken link, changed reference to foobar.

Commit:
metadata cleanup for Functions

BrucePerlerMS committed:
add options to Note

BYHAM committed:
More formatting.

BrucePerlerMS committed:
typo

BrucePerlerMS committed:
typo fix

Commit:
jairotoc5

curtand committed:
chunkified

Commit:
fixes

BrucePerlerMS committed:
formatting ...

Commit:
metadata for postgresql

SnehaGunda committed:
Updating toc

BrucePerlerMS committed:
tweak wording

v-thepet committed:
Merge pull request #19751 from jeffgilb/jeff822

Adding assign licenses quickstart

Commit:
metadata update for postgresql

BYHAM committed:
Acrolinx changes.

tynevi committed:
Update CODEOWNERS

Commit:
metadata cleanup for mvc

Commit:
Merge pull request #19758 from kriscrider/metadata_postgresql_8.21

metadata for postgresql

Lisaco88 committed:
Merge pull request #19667 from singhkays/master

migration updates

Commit:
metadata cleanup for postgresql

ShawnJackson committed:
Update hdinsight-troubleshoot-hbase.md

Commit:
Merge pull request #19759 from kriscrider/metadata_postgresql2_8.21

metadata update for postgresql

Commit:
Merge pull request #19753 from BYHAM/patch-13

SQL DW develop concurrency, formatting changes.

curtand committed:
refined

Commit:
Fixing title

ShawnJackson committed:
Update hdinsight-troubleshoot-hdfs.md

Commit:
Merge pull request #19762 from singhkays/master

Fixing title

v-thepet committed:
Merge pull request #19756 from MarkusVi/jairotoc5

jairotoc5

v-thepet committed:
Merge pull request #19754 from kriscrider/metadata_functions3

metadata cleanup for Functions

curtand committed:
moved up that diagram!

BrucePerlerMS committed:
clearify options

BrucePerlerMS committed:
clearer yet

ShawnJackson committed:
Update hdinsight-troubleshoot-storm.md

Lisaco88 committed:
Merge pull request #18256 from alkohli/SSGU5

StorSimple Garda Update 5 release notes, apply updates - go live 8/22, 10 AM

Commit:
ca78

Lisaco88 committed:
Merge pull request #19691 from SnehaGunda/envupdates

Envupdates

Commit:
Merge pull request #19761 from kriscrider/metadata_postgresql4_8.21

metadata cleanup for postgresql

robinsh committed:
more changes from Kumail

v-thepet committed:
fix

Commit:
Merge pull request #19750 from BrucePerlerMS/brp-work

add note about output formatting and soft-delete status

vladvino committed:
Added limit concurrency policy

v-thepet committed:
Merge pull request #19760 from kriscrider/metadata_postgresql3_8.21

metadata cleanup for mvc

ShawnJackson committed:
Merge pull request #19269 from ktoliver/TFS-71944

edit pass: TFS 71944

v-thepet committed:
Merge pull request #19769 from MarkusVi/ca78

ca78

Commit:
ca79

Commit:
Merge pull request #19770 from vladvino/patch-1

Added limit concurrency policy

Commit:
ca79

Commit:
Merge pull request #19772 from v-thepet/fixlinks

production bug: fix links

ShawnJackson committed:
Update application-gateway-customize-waf-rules-cli.md

Commit:
Merge pull request #19749 from cjgronlund/ml-fix

AI added for context

curtand committed:
simplifying

ShawnJackson committed:
Update application-gateway-customize-waf-rules-portal.md

ShawnJackson committed:
Update application-gateway-customize-waf-rules-powershell.md

curtand committed:
shortening

Commit:
devmgm04

Lisaco88 committed:
Merge pull request #19771 from MarkusVi/ca79

ca79

iainfoulds committed:
Freshness updates

ShawnJackson committed:
Update application-gateway-customize-waf-rules-cli.md

ShawnJackson committed:
Update application-gateway-customize-waf-rules-powershell.md

v-thepet committed:
fix more broken links

Commit:
Merge pull request #19774 from MarkusVi/devmgm04

devmgm04

SnehaGunda committed:
Updating link

iainfoulds committed:
Revisions

Commit:
freshness09

Commit:
Merge pull request #19777 from v-thepet/fixlinks

production bug: fix more broken links

ShawnJackson committed:
Merge pull request #19679 from LoriMerrick/application-gateway-customize-waf-rules-batch1

edit request: application gateway customize waf rules batch1

Ja-Dunn committed:
Merge pull request #19241 from ShawnJackson/security-azure-log-integration

edit pass: three security-azure-log-integration articles

v-thepet committed:
Merge pull request #19734 from curtand/data0821

Acrolinx one file for data-driven improvements

Commit:
Merge pull request #19780 from MarkusVi/freshness09

freshness09

Lisaco88 committed:
Merge pull request #19736 from robinsh/robin-0821

Make changes req by Kumail.

Commit:
Merge pull request #19779 from SnehaGunda/envupdates

Updating link

masnider committed:
updating guidance

Commit:
rep04

Commit:
fixing ownership changes

Commit:
Merge pull request #19782 from masnider/safety_guidance

updating guidance for durability levels

Commit:
Merge pull request #19784 from dkkapur/master

fixing ownership changes

Commit:
rep06

Commit:
Merge pull request #19785 from MarkusVi/rep06

rep06

Lisaco88 committed:
Merge pull request #19776 from iainfoulds/vmfreshness082117

VM freshness 082117

vikramdesai01 committed:
Add - AA optin details screenshot

vikramdesai01 committed:
Merge pull request #1 from vikramdesai01/vikramdesai01-patch-2

Add - AA optin details screenshot

Lisaco88 committed:
Merge pull request #19783 from MarkusVi/rep04

rep04

aliuy committed:
Acrolinx Police

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ci012

vikramdesai01 committed:
Merge pull request #2 from MicrosoftDocs/master

Bringing fork upto date

vikramdesai01 committed:
AA allow access to invoice screenshot

vikramdesai01 committed:
Merge pull request #3 from vikramdesai01/vikramdesai01-patch-3

AA allow access to invoice screenshot

genlin committed:
update the image

genlin committed:
fix megre blocking issues

vishnumane committed:
Modified service fabric repo distribution

Modified service fabric repo distribution trusty to xenial

Commit:
address merger comments 1

Commit:
added next steps in migration doc

Commit:
Added entry in TOC

Commit:
Added switcher in get-started java page

Commit:
Update for change in maven group and artifact name

Commit:
Merge pull request #19793 from vishnumane/patch-1

Modified service fabric repo distribution

Lisaco88 committed:
Merge pull request #19713 from sayantancs/ss_tp10

Doc changes for TP10 - Java SDK removal, moving all Yeo tools to NPM, mac changes

iainfoulds committed:
Format updates

Lisaco88 committed:
Merge pull request #19453 from aliuy/master

Azure Cosmos DB + Azure Functions (Serverless)

Lisaco88 committed:
Merge pull request #19710 from shizn/master

update images

Commit:
Merge pull request #19802 from iainfoulds/vmnsgcleanups

VM NSG cleanups

Lisaco88 committed:
Merge pull request #19712 from genlin/ci012

boot-diagnostics.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Lisaco88 committed:
Merge pull request #19789 from vikramdesai01/master

Add file for AA optin changes

v-alje committed:
Merge pull request #19808 from MicrosoftDocs/FromPublicRepo

From public repo
